messages/proto/messages.proto

```diff
@@ -75,13 +75,20 @@ message CommitMessage {
 message RoundChangeMessage {
   // lastPreparedProposedBlock is the last block
   // to reach Q(N) - 1 PREPARE messages
-  bytes lastPreparedProposedBlock = 1;
+  ProposedBlock last_prepared_proposed_block = 1;
 
   // latestPreparedCertificate is the PC that accompanies
   // the last proposal
   PreparedCertificate latestPreparedCertificate = 2;
 }
 
+// ProposedBlock is a (EB, r) tuple where EB is the proposed (ethereum) block
+// and r the round in which the block was constructed
+message ProposedBlock {
+  bytes block = 1;
+  uint64 round = 2;
+}
```

Comment:
This should be renamed to `Proposal`.

Ethereum, or any other blockchain doesn't have anything to do with this structure

Reply:
Renamed

[5d46bbd](https://github.com/madz-lab/go-ibft/pull/10/commits/5d46bbd7053b2c729c925c23a55d4d6d142cd166)